app/helpers/application_helper.rb

```diff
@@ -1,2 +1,17 @@
 module ApplicationHelper
+  def after_sign_in_path_for(resource)
```

Comment:
is this method duplicated on the application helper and the application controller?